python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -62,7 +63,8 @@ def test_groupby_basic(series, aggregation, pdf):
 
     check_dtype = aggregation != "count"
 
-    expect = getattr(gdf_grouped, aggregation)()
+    with expect_warning_if(aggregation == "collect"):
+        expect = getattr(gdf_grouped, aggregation)()
```

Comment:
Okay - I reviewed this a bit too quickly. I now see that we are really only deprecating the `collect` method - `agg("collect")` is still allowed. Is that correct?

Reply:
`agg("collect")` will error, `agg(list)` is the replacement

Reply:
Oh okay, I see now. It will take a bit of work to remove `agg("collect")`. Right now, doing `ddf.agg(list)` will ultimately use `agg("collect")` under the hood.

Reply:
Thanks for catching that, made all the changes necessary.